client/tokenized-express-checkout/event-handlers.js

```diff
@@ -66,8 +93,7 @@ export const onConfirmHandler = async (
 	elements,
 	completePayment,
 	abortPayment,
-	event,
-	order = 0 // Order ID for the pay for order flow.
```

Comment:
The `order` argument is no longer needed. The pay-for-order flow will just place the order with the `cartApi`, which has already knowledge of the order id in question (see its initialization logic in `client/tokenized-express-checkout/index.js`)